pep-0639.rst

```diff
@@ -1,543 +1,2209 @@
 PEP: 639
-Title: Metadata for Python Software Packages 2.2
+Title: Improving License Clarity with Better Package Metadata
 Version: $Revision$
 Last-Modified: $Date$
 Author: Philippe Ombredanne <pombredanne at nexb.com>
 Sponsor: Paul Moore <p.f.moore at gmail.com>
-BDFL-Delegate: Paul Moore <p.f.moore at gmail.com>
+PEP-Delegate: Paul Moore <p.f.moore at gmail.com>
 Discussions-To: https://discuss.python.org/t/2154
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 15-Aug-2019
-Python-Version: 3.x
 Post-History:
-Replaces: 566
 Resolution:
 
 
 Abstract
 ========
 
-This PEP describes the changes between versions 2.1 and 2.2 of the `Core
-Metadata Specification` [#cms]_ for Python packages. Version 2.1 is specified in
-PEP 566.
+This PEP defines a specification for how licenses are documented in the
+`core metadata <#coremetadataspec_>`_,
+with `license expression strings <Add License-Expression field_>`_ using
+`SPDX identifiers <#spdxid_>`_ in a new ``License-Expression`` field.
+This will make license declarations simpler and less ambiguous for
+package authors to create, end users to read and understand, and
+tools to programatically process.
 
-The primary change introduced in this PEP updates how licenses are documented in
-core metadata via the ``License`` field with license expression strings using
-SPDX license identifiers [#spdxlist]_ such that license documentation is simpler
-and less ambiguous:
+The PEP also:
 
-- for package authors to create,
-- for package users to read and understand, and,
-- for tools to process package license information mechanically.
+- `Formally specifies <Add License-File field_>`_ a new ``License-File`` field,
+  and defines how license files should be
+  `included in distributions <License files in project formats_>`_,
+  as already used by Wheel and Setuptools.
```

Comment:
Thanks for bringing up this potential issue. I address this in the `Terminology` section of this PEP:

> **Wheel Format** *(Short: wheel, Rel: Wheel project)*
>     Here, **wheel**, the standard built distribution format introduced in PEP 427
>     and `specified by PyPA <#wheelspec_>`_, will be referred to in lowercase,
>     while the `Wheel project <#wheelproject_>`_, its reference implementation,
>     will be referred to as **Wheel** in Title Case.

I originally intended to always refer to the Wheel *project* as such ("Wheel project") to disambiguate, but it ended up resulting in rather poor phrasing in many contexts, particularly given it was usually paired with Setuptools, and calling one a project but not the other would break parallelism. However, to further clarify, what I can do is append project to the combined pair, like so:

```suggestion
  as already used by the Wheel and Setuptools projects.
```

I've gone through the PEP and applied this change to every instance where the Wheel project is referenced, including here (except where it is already explicitly specified to be a "project", "implementation" or "tool"), which should address this concern.